x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/connectors.ts

```diff
@@ -19,7 +19,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
   const supertest = getService('supertest');
   const objectRemover = new ObjectRemover(supertest);
 
-  describe('Connectors', function () {
+  describe('Connectors xxx', function () {
```

Comment:
:)

```suggestion
  describe('Connectors', function () {
```

Reply:
I hate those leftovers :) removed. 
Thanks for pointing out.